<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listWidget">
     <item>
      <property name="text">
       <string>One</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Two</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="list_items_button">
     <property name="text">
      <string>List Items</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="add_item_button">
     <property name="text">
      <string>Add item</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
